src/mlpack/bindings/R/print_R.cpp

```diff
@@ -110,7 +110,7 @@ void PrintR(const util::ProgramDoc& programInfo,
   cout << "#' @details" << endl;
   cout << "#' ";
   cout << util::HyphenateString(programInfo.documentation(), "#' ") << endl;
-  cout << "#' @author" << endl;
+  cout << "#'\n#' @author" << endl;
```

Comment:
Maybe it would be easier to just add another `endl` after `HyphenateString`?  i.e.

```
cout << util::HyphenateString(programInfo.documentation(), "#' ") << endl << endl;
```

or similar.  Realistically both of these function just the same, but I think there's something nice about consistency of always using `endl` or always using `\n`, but not mixing both.  Just my opinion. :smile: